src/plugins/explore/public/application/utils/state_management/selectors/index.ts

```diff
@@ -113,10 +113,11 @@ export const selectRows = createSelector([selectResults], (results) => {
 });
 
 export const selectTotalHits = createSelector([selectResults], (results) => {
-  if ((results as any)?.hits?.total?.value !== undefined) {
-    return (results as any).hits.total.value;
-  }
-  return 0;
+  return results?.hits?.total ?? 0;
```

Comment:
would there be a `total.value`? I know `total` is a number in type, but I've seen `total.value` appearing in some cases before